<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>456</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RiskGame</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>981</width>
      <height>402</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>200</height>
          </size>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>169</width>
            <height>198</height>
           </rect>
          </property>
          <widget class="TerritoryInfoDisplay" name="territoryInfoLabel">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>171</width>
             <height>201</height>
            </rect>
           </property>
           <property name="text">
            <string></string>
           </property>
          </widget>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QTextBrowser" name="logger"/>
       </item>
       <item>
        <widget class="QPushButton" name="endTurnButton">
         <property name="text">
          <string>Zakończ turę</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="MapView" name="graphicsView">
       <property name="minimumSize">
        <size>
         <width>800</width>
         <height>400</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionCreateGAme"/>
    <addaction name="actionJoinGame"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="MyStatusBar" name="statusBar"/>
  <action name="actionCreateGAme">
   <property name="text">
    <string>Create New Game</string>
   </property>
  </action>
  <action name="actionJoinGame">
   <property name="text">
    <string>Join Existing Game</string>
   </property>
  </action>
  <action name="actionZako_cz_ture">
   <property name="text">
    <string>Zakończ ture</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MapView</class>
   <extends>QGraphicsView</extends>
   <header location="global">mapview.h</header>
  </customwidget>
  <customwidget>
   <class>MyStatusBar</class>
   <extends>QStatusBar</extends>
   <header location="global">mystatusbar.h</header>
  </customwidget>
  <customwidget>
   <class>TerritoryInfoDisplay</class>
   <extends>QLabel</extends>
   <header location="global">territories/territoryinfodisplay.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
